packages/client/src/links/internals/httpUtils.ts

```diff
@@ -223,12 +228,18 @@ export async function fetchHTTPResponse(opts: HTTPRequestOptions) {
   });
 }
 
-export async function httpRequest(
+export async function jsonHttpRequest(
```

Comment:
confusing naming when this retuns both types?

Reply:
lol yeah 🤣
All the utility functions here are a mess